### ISSUE TITLE
Overlapping selects...

### ISSUE THREAD

**Issue by events-jonas-chrisw** [JedWatson/react-select#2958]
If I use multiple  react-select component on the same page they open up behind one another.

No amount of z-index fiddling has fixed this. Even if i set the z-index of the menu to be greater than teh component it is overlapping it still sites behind it

![image](https://user-images.githubusercontent.com/33231747/44405410-1222ba80-a551-11e8-9c39-bee415c52c32.png)

Is there any solution?

**Comment by pbortnick** [comment]
@events-jonas-chrisw  Question - were you using multiple sections within one dropdown/search ? That's exactly what I'm looking to do. Or something else entirely?

**Comment by events-jonas-chrisw** [comment]
no, two separate dropdowns. the top one by default sits behind behind the one below. I had to fudge them into their own containers and explicitly set the z-index on them.

**Comment by therealtgd** [comment]
+1 on this issue. Using multiple dropdowns is a nightmare because they overlap.
